src/transform-type.ts

```diff
@@ -97,3 +100,7 @@ function transformFields(ty: IdlFieldsType) {
   }
   return ty
 }
+
```

Comment:
Assuming this comment is for later?

Reply:
Yep, we could take it out, but it's just a comment to remind us :)